client/api.go

```diff
@@ -28,15 +34,16 @@ import (
 	"github.com/stripe/stripe-go/v72/feerefund"
 	"github.com/stripe/stripe-go/v72/file"
 	"github.com/stripe/stripe-go/v72/filelink"
-	"github.com/stripe/stripe-go/v72/identity/verificationreport"
-	"github.com/stripe/stripe-go/v72/identity/verificationsession"
+	identityverificationreport "github.com/stripe/stripe-go/v72/identity/verificationreport"
+	identityverificationsession "github.com/stripe/stripe-go/v72/identity/verificationsession"
 	"github.com/stripe/stripe-go/v72/invoice"
 	"github.com/stripe/stripe-go/v72/invoiceitem"
-	"github.com/stripe/stripe-go/v72/issuing/authorization"
+	issuingauthorization "github.com/stripe/stripe-go/v72/issuing/authorization"
 	issuingcard "github.com/stripe/stripe-go/v72/issuing/card"
-	"github.com/stripe/stripe-go/v72/issuing/cardholder"
+	issuingcardholder "github.com/stripe/stripe-go/v72/issuing/cardholder"
```

Comment:
are those changes breaking changes?

Reply:
This was actually merged in https://github.com/stripe/stripe-go/pull/1323 .